aiida/engine/processes/calcjobs/manager.py

```diff
@@ -153,14 +174,14 @@ def request_job_info_update(self, job_id):
             pass
 
     def _ensure_updating(self):
-        """
-        Ensure that we are updating the job list from the remote resource.
+        """Ensure that we are updating the job list from the remote resource.
```

Comment:
this is not only ensuring things but also performing the update as well, right?
should this be called something like `_update_loop` ?

Reply:
I don't think so, the actual updating is happening in `_update_job_info`. This `_ensure_updating` is really just making sure that, each time it is called, a callback has already been registered with the appropriate delay, to perform the updating. If not, it schedules it, if is already scheduling, it is a no-op